Help2guides (#2373)
openmicroscopy/ome-documentation: omero/users/clients-overview.rst

computer (this can easily be installed from `<https://java.com/>`_ if it is not
already included in your OS).

Our :help:`User guides<>` provide a series of
workflow-based guides to performing common actions in the client applications,

> such as importing and viewing data, exporting images and using the measuring

Reviewer: ```suggestion
such as importing and viewing data, exporting images, drawing and analyzing of ROIs.
```

> tool. 

Reviewer: ```suggestion
 
```
